EIPS/eip-4844.md

```diff
@@ -0,0 +1,554 @@
+
+The block-by-block behavior is roughly as follows.
+If in block `N`, `blob_gas = G1`, and block `N` has `X` blobs, then in block `N+1`, `excess_blobs` increases by `X - TARGET_BLOB_TXS_PER_BLOCK`,
+and so the `blob_gas` of block `N+1` increases by a factor of `2**((X - TARGET_BLOB_TXS_PER_BLOCK) / GASPRICE_UPDATE_FRACTION_PER_BLOB)`.
+Hence, it has a similar effect to the existing EIP 1559, but is more "stable" in the sense that it responds in the same way to the same total usage regardless of how it's distributed.
+
+## Backwards Compatibility
+
+### Blob non-accessibility
+
+This EIP introduces a transaction type that has a distinct mempool version (`BlobTransactionNetworkWrapper`) and execution-payload version (`SignedBlobTransaction`),
+with only one-way convertibility between the two. The blobs are in the `BlobTransactionNetworkWrapper` and not in the `SignedBlobTransaction`;
+instead, they go into the `BeaconBlockBody`. This means that there is now a part of a transaction that will not be accessible from the web3 API.
+
+### Mempool issues
+
+Blob transactions are unique in that they have a variable intrinsic gas cost. Hence, a transaction that could be included in one block may be invalid for the next.
+To prevent mempool attacks, we recommend a simple technique: only propagate transactions whose `gas` is at least twice the current minimum.
+
+Additionally, blob transactions have a large data size at the mempool layer, which poses a mempool DoS risk,
+though not an unprecedented one as this also applies to transactions with large amounts of calldata.
+The risk is that an attacker makes and publishes a series of large blob transactions with fees `f9 > f8 > ... > f1`,
+where each fee is the 10% minimum increment higher than the previous, and finishes it off with a 21000-gas basic transaction with fee `f10`.
+Hence, an attacker could impose millions of gas worth of load on the network and only pay 21000 gas worth of fees.
+
+We recommend a simple solution: both for blob transactions and for transactions carrying a large amount of calldata,
+increase the minimum increment for mempool replacement from 1.1x to 2x, decreasing the number of resubmissions an attacker can do at any given fee level by ~7x.
+
+## Test Cases
+
+TBD
+
+## Security Considerations
+
+This EIP increases the storage requirements per Beacon block by a maximum of ~2 MB.
+This is equal to the theoretical maximum size of a block today (30M gas / 16 gas per calldata byte = 1.875M bytes), and so it will not greatly increase worst-case bandwidth.
+Post-merge, block times are expected to be static rather than an unpredictable Poisson distribution, giving a guaranteed period of time for large blocks to propagate.
+
+The _sustained_ load of this EIP is much lower than alternatives (eg. [EIP-4488](./eip-4488.md) and [EIP-4490](./eip-4490.md)),
```

Comment:
Any EIPs you link to are considered "dependencies", and thus require those EIPs be as far along in the EIP process as this one.  `EIP-4490` isn't an EIP at all, and `EIP-4488` is a draft I think (and may not reach final before this).  I recommend just describing (very briefly) the alternatives and then including additional links out in the discussions-to thread.

As mentioned previously, EIPs should stand on their own aside from their dependencies, and the EIP document isn't an appropriate place to make comparisons against other proposals aside from a high level, "we could do X, but we decided this is superior because of Y".